lib/rubocop/cop/rspec/factory_bot/create_list.rb

```diff
@@ -51,26 +52,19 @@ def on_block(node)
             return unless n_times_block_without_arg?(node)
             return unless contains_only_factory?(node.body)
 
-            add_offense(node.send_node, message: MSG_CREATE_LIST)
+            add_offense(node.send_node, message: MSG_CREATE_LIST) do |corrector|
+              CreateListCorrector.new(node.send_node).call(corrector)
+            end
           end
 
           def on_send(node)
             return unless style == :n_times
 
             factory_list_call(node) do |_receiver, _factory, count, _|
```

Comment:
it seems that we never use the other captures, so we can just remove them from the pattern

Reply:
I’ll fix that in a separate PR.

Reply:
See #959.